src/lib/eZ/RichText/Resources/stylesheets/docbook/xhtml5/edit/core.xsl

```diff
@@ -688,7 +688,17 @@
         <xsl:value-of select="@ezxhtml:class"/>
       </xsl:attribute>
     </xsl:if>
-    <xsl:if test="@ezxhtml:align">
+    <xsl:if test="@xml:id">
+      <xsl:attribute name="id">
+        <xsl:value-of select="@xml:id"/>
+      </xsl:attribute>
+    </xsl:if>
+    <xsl:if test="@ezxhtml:align and not(contains( @ezxhtml:class, 'ez-custom-tag' ))">
```

Comment:
Hmm, this is unexpected choice of condition for styles. `<eztemplate type="style">` is a custom style and this is what I expected you to rely on when I wrote 

> eztemplate of `"style"` `type`

Should've been more clear I guess, I apologize.

Reply:
@alongosz hopefully it's gonna be ok now: https://github.com/ezsystems/ezplatform-richtext/pull/166/commits/107f7371670c5f3cf0f21faa8de3aaa533971911